### ISSUE TITLE
Meta-question about eslint-plugin-es5 usage

### ISSUE THREAD

**Issue by tinymachine** [WICG/focus-visible#191]
This package's [`.eslint.js`](https://github.com/WICG/focus-visible/blob/c16758abceed478e69cad11ada4a464a1ffa17ad/.eslintrc.js) includes `eslint-plugin-es5` in `plugins`, but never sets up (or "extends") any rules (e.g. `"extends": ["plugin:es5/no-es2015"]`). Does the plugin do anything without configuring any rules? 

I tried this on a test project and the plugin failed to catch ES2015+ code without rules being configured. Just curious if I'm missing something!

**Comment by robdodson** [comment]
hmmm...It seemed to work for me just now when I tried it.

I added `const foo = 'bar'` to src/focus-visible.js, then ran `npx eslint src/focus-visible.js` and it complained about the use of `const`.

Going to close this as working as intended but happy to keep discussing in this thread if you think there's an issue.

**Comment by tinymachine** [comment]
Thanks so much for taking the time to test it out and replying! For me, ESLint complains about `const` even without any plugins configured: `Parsing error: The keyword 'const' is reserved`.

When I try the following code (via [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)) using the ES6 `Object.assign()` method with just `"plugins": ["es5"]` configured in my .eslintrc.js, ESLint doesn't complain (though we want it to):

```js
var target = { a: 1, b: 2 };
var source = { b: 4, c: 5 };
var returnedTarget = Object.assign(target, source);
```
 
When I add `"extends": ["plugin:es5/no-es2015"]` to my .eslintrc.js, it complains appropriately: `ES6 static methods not allowed: Object.assign`. 

Thanks again for engaging, and no worries if you're too busy to follow up!

**Comment by robdodson** [comment]
oh interesting. ok maybe I have it configured wrong. Let me reopen the ticket and I'll do a PR to add the extends field.

**Comment by robdodson** [comment]
by the way, if you're interested at all in submitting a pull request for this I'd be happy to help you land it. No worries if you're too busy, but wanted to offer just in case that sounds interesting :)

**Comment by tinymachine** [comment]
Sure, I'd be happy to -- should be really quick...

**Comment by tinymachine** [comment]
Hmm - [apparently](https://www.npmjs.com/package/eslint-plugin-es5#usage) the only two configurable options with eslint-plugin-es5 are `plugin:es5/no-es2015` and `plugin:es5/no-es2016`, but nothing more recent. I suppose configuring both of those is better than nothing. I'll go ahead and do that unless I hear otherwise from you.

**Comment by robdodson** [comment]
yep that sounds good. We basically want to enforce that we only are using es5

**Comment by tinymachine** [comment]
I just wonder if some ES2017+ could slip through the cracks without having presets for those ES versions...(or maybe I'm thinking about this wrong!)

**Comment by robdodson** [comment]
> I just wonder if some ES2017+ could slip through the cracks without having presets for those ES versions...(or maybe I'm thinking about this wrong!)

Hm, maybe. I'll do my best to keep an eye on it. Thank you again :)

Fixed in #192

**Comment by tinymachine** [comment]
My pleasure -- thanks for sharing this library, and for all your a11y work.
